### PR TITLE
feat: return json data instead of wrapped data in sessions and event grpc endpoints

### PR DESCRIPTION
**Pull request by itayd**
caller (usually ui) need to specify `json_values` in request.

Refs: ENG-1403

### CODE REVIEW

File: sdk/sdktypes/value.go
Comment (daabr) [inline sdk/sdktypes/value.go:180]:
```suggestion
	u, err := ValueWrapper{SafeForJSON: true}.Unwrap(v)
```

File: proto/autokitteh/events/v1/svc.proto
Comment (daabr) [inline proto/autokitteh/events/v1/svc.proto]: 1. s/contained/contain/
2. Explain also what happens if the bool is `false`

File: proto/autokitteh/events/v1/svc.proto
Comment (daabr) [inline proto/autokitteh/events/v1/svc.proto]: Same as above

File: proto/autokitteh/sessions/v1/svc.proto
Comment (daabr) [inline proto/autokitteh/sessions/v1/svc.proto]: Same as in `proto/autokitteh/events/v1/svc.proto`

**Review by daabr — APPROVED** [review]
If you need to specify a bool in the request that affects the "mode" of returned value, and there's no obvious way to understand the response without that bool - this is a strong sign of unnecessary complexity and a code smell.

It's not blocking this PR, but it needs to be improved.

Before merging, see the minor suggestions.

**Comment by itayd** [comment]
> If you need to specify a bool in the request that affects the "mode" of returned value, and there's no obvious way to understand the response without that bool - this is a strong sign of unnecessary complexity and a code smell.

Yup - we are doing this as a shortcut to achieve a very specific goal. In the future we'll think of something better.

File: proto/autokitteh/events/v1/svc.proto
Comment (itayd) [reply proto/autokitteh/events/v1/svc.proto]: done

File: proto/autokitteh/sessions/v1/svc.proto
Comment (itayd) [reply proto/autokitteh/sessions/v1/svc.proto]: refed

File: proto/autokitteh/events/v1/svc.proto
Comment (itayd) [reply proto/autokitteh/events/v1/svc.proto]: refed

**Pull request merged**
